packages/server/src/service/trpc.test.ts

```diff
@@ -0,0 +1,61 @@
+import { sampleStateInfo } from './letter'
+import { mocked } from 'ts-jest/utils'
+import { VbmRpc } from './trpc'
+import mailgun, { Mailgun } from 'mailgun-js'
+import twilio, { Twilio } from 'twilio'
+import { Request } from 'express'
+import { toContactMethod, StateInfo } from '../common'
+import { sendFaxes as unmockedSendFaxes } from './twilio'
+import { isE164 } from '../common/util'
+
+// To avoid doing actual requests (and setting a lot of new envs for tests),
+// we mock the majority of functions/modules related to trpc.
+
+jest.mock('mailgun-js')
+jest.mock('twilio')
+jest.mock('./twilio')
+jest.mock('./trpc')
+jest.mock('../common', () => {
+  const common = jest.requireActual('../common')
+  common.processEnvOrThrow = jest.fn().mockImplementation(key => {
+    const value = process.env[key]
+    if (value) return value
+    console.warn(`The value for process.env[${key}] was not defined, no errors were thrown to complete the check. Please check if this was intentional.`)
+    return 'NO_VALUE'
+  })
+  return common
+})
+
+// mailgun & twilio
+mocked(mailgun).mockReturnValue(null as unknown as Mailgun)
+mocked(twilio).mockReturnValue(null as unknown as Twilio)
+const sendFaxes = mocked(unmockedSendFaxes).mockResolvedValue([])
+
+// trpc
+const trpc = mocked(new VbmRpc())
+mocked(trpc).register = jest.fn().mockImplementation(async (info: StateInfo) => {
```

Comment:
We don't want to mock `register`.  We want to call the real `register`, mock everything it can call, and check if `sendFaxes` was called properly